test/server_search.cpp

```diff
@@ -1045,29 +1045,13 @@ TEST(ServerSearchTest, searchResults)
     },
 
     {
-      /* query */          "pattern=yellow&books.id=" RAYCHARLESZIMID,
-      /* start */            0,
+      /* query */          "pattern=yellow%20submarine&books.id=" RAYCHARLESZIMID,
```

Comment:
The "yellow submarine" test point should have been added instead of replacing the "yellow" test point.